apps/examples/testcase/le_tc/filesystem/tc_fs_procfs.c

```diff
@@ -94,10 +103,156 @@ static int read_dir_entries(const char *dirpath)
 
 	return ERROR;
 }
+#if !defined(CONFIG_DISABLE_MOUNTPOINT) && defined(CONFIG_FS_PROCFS)
+#ifndef CONFIG_FS_PROCFS_EXCLUDE_UPTIME
+static int procfs_uptime_ops(char *dirpath)
+{
+	int ret;
+	int	fd;
+	int fd1 = 0;
+	struct stat st;
+	fd = open(PROC_UPTIME_PATH, O_RDONLY);
+	if (fd < 0) {
+		printf("Failed to open \n" );
+		return ERROR;
+	}
+
+	ret = stat(PROC_UPTIME_PATH, &st);
+	if (ret != OK) {
+		printf("failed to stat \n");
+		return ERROR;
+	}
+	ret = dup2(fd, fd1);
```

Comment:
any specific reason that testing dup2 here?

Reply:
For covering uptime_dup operation